src/SqlAsyncCollector.cs

```diff
@@ -171,15 +174,25 @@ private async Task UpsertRowsAsync(IEnumerable<T> rows, SqlAttribute attribute,
                 ObjectCache cachedTables = MemoryCache.Default;
                 var tableInfo = cachedTables[cacheKey] as TableInformation;
 
+                int timeout = DEFAULT_TABLE_INFO_CACHE_TIMEOUT_MIN;
+                string timeoutEnvVar = Environment.GetEnvironmentVariable("AZ_FUNC_TABLE_INFO_CACHE_TIMEOUT_MIN");
```

Comment:
Let's name this `AZ_FUNC_TABLE_INFO_CACHE_TIMEOUT_MINUTES` so it isn't confused with minimum